docs/product-release-notes/src/main/asciidoc/con-bxms-rn-whats-new.adoc

```diff
@@ -0,0 +1,156 @@
+
+
+== Process Engine
+
+=== Process and task administration API
+A process administration API and task administration API have been introduced to simplify some of the more complex administrator use cases.
+The process admin API enables you to:
+
+* Retrieve all process definition nodes
+* Cancel node instances
+* Retrigger node instances
+* Update the timer (absolute or relative)
+* List timer instances
+* Trigger nodes
+
+The task administration API enables you to:
+
+* Add and remove potential owners and excluded owners and business administrators
+* Add and remove task inputs and outputs
+* List, create, and cancel escalations and notifications
+
+=== Advanced task routing
+When tasks are assigned to a group of users, you can use pluggable task assignment strategies to automatically assign tasks to a suitable individual immediately. This enables more efficient task allocation, based on all properties associated with the task (for example potential owners and task priority but also task data that includes information such as geography, required skills, and so forth). You can use business rules to define the assignment logic, making it easy to customize this to your needs.
+
+=== Process execution server
+The process execution server (also known as kie-server) has been extended to support core engine features and offers a remote API for these operations. In addition the following architectural changes were introduced.
+
+==== Separate Business Central from execution server
+Business Central now delegates all of its requests to the execution server as well. The main advantage is that Business Central can now be used to monitor any set of execution servers. By linking the execution server to Business Central, the process and task monitoring UIs in Business Central can now connect to this execution server and show all relevant information. When multiple independent execution servers are used, you can either connect to a specific one or use the smart router to aggregate information across multiple servers. 
+
+==== Smart router
+The smart router (also known as kie-server-router) can be used as a proxy to help manage multiple independent process execution servers.
+
+==== Aliases
+You can now use aliases instead of container IDs in the remote REST APIs of the execution server. For more information, see the "Editing container aliases through Business Central" section of the _{PRODUCT} User Guide_.
+ 
+=== Business Central Monitoring web application
+A new distribution `war` file that enables you to build custom cloud images has been added in this release.The Business Central Monitoring web application should be used with Decision Server instances to manage all run time capabilities in the cloud. This includes managing containers, process instances, tasks, and dashboards. Unlike the standard Business Central distribution, no authoring capabilities are offered.
```

Comment:
There are quite a few problems here but github ate my comments somehow. From a missing space to passive voice to an ambiguous "this" which is also against IBM Style, as is "should". Also "runtime" not "run time", and also "offer" was not really suitable here. Suggested fix:

+A new distribution `war` file that enables you to build custom cloud images has been added in this release. Use the Business Central Monitoring web application with Decision Server instances to manage containers, process instances, tasks, dashboards, and other runtime capabilities in the cloud. Unlike the standard Business Central distribution, no authoring capabilities are available.